### PR TITLE
feat: remove wallet connect 1.0

### LINKED ISSUE #14978: remove WalletConnect v1.0
WalletConnect v1.0 is shutting down on March 1st

### PR DESCRIPTION
**Pull request by briansztamfater**
fixes #14978 

### Summary

Wallet Connect 1.0 is shutting down on March 1st, this PR removes its integration. Also, as the Wallet Connect 2.0 integration was made with the library in beta version, it has got outdated and not working anymore, so disabled session pairing until we re-enable them with #13429 implementation

### Testing

In this case smoke testing scanning wallet connect QR codes and check that don't crash and nothing happens would be fine.

status: ready

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -1960,5 +1960,6 @@
     "my-albums": "My albums",
     "images": "images",
     "only-6-images": "You can only add 6 images to your message",
-    "delivered": "Delivered"
+    "delivered": "Delivered",
+    "wallet-connect-not-implemented": "Wallet Connect not implemented"
```
Comment (flexsurfer) [inline translations/en.json]: is it like only for us or for the release app as well? not sure it should be translated

**Review by flexsurfer — APPROVED** [review]

File: translations/en.json (hunk 1 above)
Comment (briansztamfater) [reply translations/en.json]: Only for us I think, we need to upgrade and re-enable WC 2.0 for the MVP afaik, so this alert shouldn't see the light. I can remove the translation and use a plain string.

**Review by yqrashawn — APPROVED** [review]

**Review by siddarthkay — APPROVED** [review]
🥲

**Comment by flexsurfer** [comment]
@briansztamfater could you pls resolve conflicts? and lets merge it, im not sure it need any manual testing

**Comment by briansztamfater** [comment]
@flexsurfer thanks for merging, I did a quick testing locally and seems that nothing was broken with these changes

**Pull request merged**